test/parallel/test-child-process-exec-error.js

```diff
@@ -3,6 +3,11 @@ var common = require('../common');
 var assert = require('assert');
 var child_process = require('child_process');
 
+if (common.isWindows) {
+  common.skip('symlinks are weird on windows');
+  return;
+}
```

Comment:
Why is the above check done? The test does not use symlinks. (If there is a good reason to skip the test, then the check for `common.isWindows` below can be removed.)